manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Stellar.StellarCLI
+PackageVersion: 22.0.0-rc.1
+PackageLocale: en-US
+Publisher: Stellar
```

@fnando I think the publisher would be the full foundation's name.
```suggestion
Publisher: Stellar Development Foundation
```